### DISCUSSION TITLE
Which ESP32 hardware to use?

### DISCUSSION THREAD

**Discussion by Slion** [fsaris/EspHome-AwoX-BLE-mesh-hub#90] [General]
I understand I need to connect an ESP32 device to my Home Assistant over USB to get this to work. Which ESP32 device do you use and recommend?

**Comment by fsaris** [comment]
I'm using a ESP-WROOM-32 development board

**Reply by Kees48** [reply]
Ik heb ESP32-WROOM-32 gekocht. Ik kan daar nix mee. Zal ik dat terugsturen of kan ik daar nog een bordje bijkopen (wat waar?) waar de module op past. Ik wil graag mijn Awox lampen en Plug stekkers weer aan de praat krijgen in home assistant. Groet Kees

**Comment by alka79** [comment]
I use a MINI D1 ESP32 board from wemos (possibly a clone). It is ESP32 WROOM. 
Nice and small, it works like a charm.

**Comment by michalpulda** [comment]
I use [LaskaKit ESP32-LPKit](https://www.laskakit.cz/laskakit-esp32-lpkit-pcb-antenna/?variantId=10488) with external [antenna](https://www.laskakit.cz/anteny/) (it needs a [programmer](https://www.laskakit.cz/laskakit-ch9102-programmer-usb-c--microusb--uart/), though). They [ship](https://www.laskakit.cz/doprava-a-platba/) mainly within EU.
